.github/ci/rtthread-littlefs/build.sh

```
@@ -0,0 +1,1507 @@
+    rm -f "$qemu_pid_file"
+
+    setsid -w sh -c '
+        printf "%s\n" "$$" > "$2"
+        cd "$1" || exit 1
+        if [ -f ./qemu.sh ]; then
+            QEMU_AUDIO_DRV=none exec sh ./qemu.sh
+        else
+            exec qemu-system-arm -M vexpress-a9 -smp 2 -kernel rtthread.elf \
+                -nographic -serial mon:stdio
+        fi
+    ' sh "$bsp_dir" "$qemu_pid_file" > "$log_file" 2>&1 &
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)